config/locales/de.yml

```diff
@@ -81,6 +81,8 @@ de:
       weak_password: "Um die Sicherheit zu verbessern, sollten Sie ein komplexeres Passwort wählen"
     teams:
       not_existing: "Es existiert kein Team mit der Id %{id}"
+    folders:
+      duplicated_inbox: 'Es kann nur ein Inbox Ordner im personal Team geben'
```

Comment:
Same with this as in the swissgerman translation, maybe change it to german and else use "Personal Team"